docs: Upgrade docz to v2

lightnet328 opened the pull request:
# WHAT

- Upgrade docz to v2
- Create docz package in docs dir
- Remove an unused config

# WHY

- The docz package is for using "Shadowing`
  - https://www.gatsbyjs.org/docs/themes/shadowing/

![localhost_3000_(iPad Pro)](https://user-images.githubusercontent.com/2351326/71826390-25e8a580-30e1-11ea-9d1e-997648fca1fc.png)

lightnet328 commented on package.json:
```diff
@@ -62,17 +62,20 @@
     "enzyme-adapter-react-16": "1.15.2",
     "eslint": "6.8.0",
     "eslint-config-prettier": "6.9.0",
+    "eslint-loader": "3.0.3",
```
```
warning “docz > gatsby-plugin-eslint@2.0.8” has unmet peer dependency “eslint-loader@^3.0.2".
```
For resolving this.

lightnet328 commented:
Thanks for your review! I fixed issues caused by created a new package for docz.

rot1024 commented:
Thank you for great work! But I want you to fix more following points:

## 1. Remove extra heading on component pages

> name: Viewer route: /components/Viewer
> 
> menu: Components

You can fix `docs/scripts/generateDoc.js` to achieve it.

![image](https://user-images.githubusercontent.com/3888696/72333847-d79a5e80-36ff-11ea-9339-778dd4d1def8.png)

## 2. Tweak styles of tables

Adjust the styles of the table to be closer to the previous document.

![image](https://user-images.githubusercontent.com/3888696/72334229-50011f80-3700-11ea-90a4-618e4963f17f.png)

to

![image](https://user-images.githubusercontent.com/3888696/72334593-e46b8200-3700-11ea-980a-daf39a9cd41b.png)

## 3. Add background color to `<code>` block, and tweak text font styles

![image](https://user-images.githubusercontent.com/3888696/72334887-678cd800-3701-11ea-806f-e8a33c1c9126.png)

to

![image](https://user-images.githubusercontent.com/3888696/72334852-5c39ac80-3701-11ea-9fef-32246266ffa4.png)

lightnet328 commented:
Thanks for reviewing! I've just changed. Please review it again.

- Table and InlineCode styles
  - imported from current styles and adjusted it for light mode
- Fix generateDoc.js

rot1024 commented:
Excellent. Thanks to you, the appearance of the document has become more beautiful.

Sadly, the following issue has left:

Storybook on docz does not work because `/cesium/Widgets/widgets.css` and `/cesium/Cesium.js` are not found.

Please change webpack config of docz to copy Cesium files and set env var on build.

Reference:

```js
  modifyBundlerConfig: config => ({
    ...config,
    // for storybook
    externals: {
      ...config.externals,
      cesium: "Cesium",
    },
    // for storybook
    plugins: [
      ...config.plugins,
      new webpack.DefinePlugin({
        CESIUM_BASE_URL: JSON.stringify("/cesium"),
      }),
      new CopyPlugin([
        {
          from: "node_modules/cesium/Build/Cesium",
          to: "cesium",
        },
      ]),
    ],
  }),
```

lightnet328 commented:
I’m so sorry to bother you many times. I added `gatsby-node.js` to copy `cesium` modules.

rot1024 commented on package.json:
```diff
@@ -13,24 +13,21 @@
-    "src",
-    "dist"
-  ],
+  "files": ["src"],
```
```suggestion
  "files": [
    "src",
    "dist"
  ],
```
